include/acl_types.h

```diff
@@ -295,6 +296,15 @@ typedef struct host_op_struct {
   size_t m_size_sent;
 } host_op_t;
 
+struct sideband_signal_t {
+  unsigned
+      port_identifier; // 0 is data, 1, 2, 3, etc. are for different sideband
+                       // signals. Defined in the compiler
+                       // acl/llvm-project/llvminclude/llvm/Analysis/FPGAAnalysis/Utils/StreamParameters.h
```

Comment:
I think this is fine since we also reference some path in the acl_printf.cpp top. I am going to change this to a relative path in the enum definition.